toolchain/check/diagnostic_emitter.h

```diff
@@ -57,6 +57,9 @@ class DiagnosticEmitter : public DiagnosticEmitterBase {
                         ContextFnT context_fn) const
       -> Diagnostics::ConvertedLoc;
 
+  // Adds `in import` note.
+  static auto InImport(Diagnostics::Loc loc, ContextFnT context_fn) -> void;
```

Comment:
I think it'd help to include a verb in the name

```suggestion
  static auto AddInImport(Diagnostics::Loc loc, ContextFnT context_fn) -> void;
```

Reply:
Done.